qt/scientific_interfaces/General/StepScan.cpp

```diff
@@ -598,7 +598,8 @@ void StepScan::generateCurve(const QString &var) {
 
     MatrixWorkspace_sptr top = AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>(m_plotWSName);
     MatrixWorkspace_sptr bottom = norm->getProperty("OutputWorkspace");
-    top = top / bottom;
+    top /= bottom;
+    AnalysisDataService::Instance().addOrReplace(m_plotWSName, top);
```

Comment:
why is this needed? I thought the `/=` would preserve the pointer 🤔

Reply:
It seems like `top = top / bottom;` creates a new workspace, which it does not put it back in AnalysisDataService